static/app/components/events/interfaces/spans/spanGroup.tsx

```diff
@@ -53,11 +54,11 @@ class SpanGroup extends React.Component<PropType, State> {
   };
 
   renderSpanChildren = () => {
-    if (!this.state.showSpanTree) {
-      return null;
-    }
-
-    return this.props.renderedSpanChildren;
+    return (
```

Comment:
I don't think this should have a functional impact on the span tree as all this does is use `display: none;` to hide the children instead of causing them to unmount. This helps to ensure that the children are mounted only once which is useful in the `componentDidMount` to scroll to the span on pageload.

cc @dashed if you have any thoughts on this

Reply:
@Zylphrex I think that can be alleviated if this lifecycle was moved to a common ancestor. See my comment here: https://github.com/getsentry/sentry/pull/25760/files#r623985043